intelmq/bots/BOTS

```diff
@@ -1,5 +1,15 @@
 {
     "Collector": {
+
+        "bitsight":{
```

Comment:
Also, please adapt the bot name (e.g. `Bitsight Alert Stream`), collector and parser name should be same/similar.